dist/common/systemd/scylla-helper.slice

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Slice used to run companion programs to Scylla. Memory, CPU and IO restricted
+DefaultDependencies=no
+Before=slices.target
+Wants=-.slice
+After=-.slice
+
+[Slice]
+MemoryAccounting=true
+IOAccounting=true
+CPUAccounting=true
+
+CPUWeight=10
+IOWeight=10
+
+# MemoryHigh is the throttle point
+MemoryHigh=4%
+# MemoryMax is the OOM point. As Scylla reserves 7% by default, the other two percent goes to
+# the kernel and other non contained processes
+MemoryMax=5%
+# Systemd deprecated settings BlockIOWeight, MemoryLimit and CPUShares. But they are still the ones used in RHEL7
```

Comment:
Seastar has the ability to use an absolute reserve:

```
struct configuration {
    optional<size_t> total_memory;
    optional<size_t> reserve_memory;  // if total_memory not specified
    optional<size_t> cpus;
    optional<cpuset> cpu_set;
    optional<unsigned> io_queues;
};
```

which unfortunately we don't use. But the Seastar's default is max(1.5GB, 7%), so the percentage calculation can be too limiting in small instances.

Perhaps the postinstall script can calculate and write an override to /etc/systemd.

Reply:
The problem with postinstall is that it is still wrong when people change instance sizes.
It's a weird operation if you think in terms of real hardware, but in the cloud it happens all the time...

But perhaps you are right and the solution for that problem is a different one: so many things change when you move between instance sizes, that the best solution is probably to have a script to re-configure.

If installing the rpm again would re-do the post install part, that's probably a good solution.

Reply:
I really wish systemd could read this from an environment variable, then we could put the reservation there. It seems like they only support environment variables in key specific directives, like `ExecStart`, but not in the slice configuration. Then I guess an override it is